logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/LoginUseCase.kt

```diff
@@ -19,6 +19,7 @@ sealed class AuthenticationResult {
     ) : AuthenticationResult()
 
     sealed class Failure : AuthenticationResult() {
+        object SocketError : Failure()
```

Comment:
this socketError is kind of error that not easy to map in head of the developer! Since it's relates to the tech of the proxy itself, isn't better to rename it to ProxyError?